app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -119,7 +119,7 @@ form {
   }
 
   /* Text Fields */
-  input[type=text], input[type=password], input[type=email], input[type=number], input[type=url], input[type=tel], textarea {
+  input[type=text], input[type=password], input[type=email], input[type=number], input[type=url], input[type=tel], input[type=date], textarea {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Each selector in a comma sequence should be on its own line